### PR TITLE
Reading list pagination improvement

### PR DESCRIPTION
**Pull request by CMasterson**
## Summary
Loads almost a full page ahead to try give users a smoother experience.

### CODE REVIEW

File: Sources/ReadingListKit/ReadingListCell.swift [hunk 1]
```diff
@@ -17,6 +17,7 @@ public struct ReadingListCell: View {
                 VStack(alignment: .leading) {
                     Text(model.title)
                         .font(.headline)
+                        .lineLimit(2)
```
Comment (CMasterson) [inline Sources/ReadingListKit/ReadingListCell.swift:20]: This is an unrelated fix for titles spanning too many lines.

**Review by dskuza — APPROVED** [review]
